requirements/base.txt

```diff
@@ -12,4 +12,6 @@ python-heatclient>=1.6.1,<1.7
 python-keystoneclient>=3.17.0,<3.18
 python-novaclient>=7.1.2,<7.2
 tenacity>=6.2
+channels==2.4.0
+pyguacamole==0.9
```

Comment:
Can we turn these two into `>=x` or, if necessary, `>=x,<y` deps? I'm not a big fan of hard-freezing dependencies.

Reply:
sure, done.